research/src/components/breadcrumb-anatomy.jsx

```diff
@@ -1,25 +1,20 @@
-/* eslint-disable */
-
 import React from 'react'
+import { AnatomyWrapper, Host, Slot, Part } from './anatomy-components'
 import './breadcrumb-anatomy.css'
 
 const BreadcrumbAnatomy = () => {
   return (
-    <div className="component-anatomy-wrapper">
-      <input type="checkbox" id="show-slots" />
-      <label htmlFor="show-slots"> Show slots</label>
-      <div className="component-anatomy">
-        <host name="breadcrumb" role="navigation">
-          <slot name="breadcrumb-container">
-            <part name="ol">
-              <part name="li [part=item]" data-slot>
-                item
-              </part>
-            </part>
-          </slot>
-        </host>
-      </div>
-    </div>
+    <AnatomyWrapper>
```

Comment:
Thanks for fixing this up!

Reply:
![image](https://user-images.githubusercontent.com/17087167/222918041-50d286cd-039c-4662-bcae-4e507f1bd4a3.png)
looks great

Reply:
![image](https://user-images.githubusercontent.com/17087167/222918166-82730a5a-7cf2-4b40-8f26-df01acfc9969.png)

It's currently broken on live, so double thanks for fixing this up!